app/api/PRMSCoverageTool.py

```diff
@@ -0,0 +1,106 @@
+__author__ = 'jerickson'
+
+import netCDF4
+import shutil
+import os
+
+### PRMS File representation
+class PRMSFile:
+    def __init__(self, basefile):
+        self.basefile = basefile
+        self.workingscenario = None
+
```

Comment:
@jerickson42 please change these `UpperCamelCase` class methods to `lower_snake_case`